pkg/pr/pr.go

```diff
@@ -126,9 +133,16 @@ func generatePRArgs(options *Options) []string {
 }
 
 func update(exe utils.Executor, branchID string, prID string) error {
+
+	//Run tests
+	err := test.RunTest(exe)
+	if err != nil {
+		return err
+	}
+
 	// Push the current branch to the already existing git remote
 	s := utils.StartSpinner("Updating Pull Request #"+prID+"...", "Pull Request #"+prID+" has been updated.")
-	_, err := exe.Command("git", "push")
+	_, err = exe.Command("git", "push")
```

Comment:
Doesn't the linter react to this? I've seen that before, at least (doesn't like "shadowing").

Reply:
I think there's an argument to be made to ignore shadowing (for err specifically) when linting

https://github.com/golangci/golangci-lint/issues/166